### PR TITLE
Added a Wheels section to the Python Build Docs

### PR DESCRIPTION
**Pull request by jagadeeshmadinni**
### Details:
 - *Included explanation about python wheels and the wheels built by OpenVINO. Updated the Linux build example to include cloning the repo that was missing earlier*

### Tickets:
 - *22017*

### CODE REVIEW

File: src/bindings/python/docs/build.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+
+## Using Python Wheels for OpenVINO™
+Wheels are portable python packages that are ready to install upon download. They are the de facto binary distributions in Python and avoid the compiling of extension modules and associated dependency issues on the user end. Thus, wheel installation is faster, smoother and makes the lives of both developers and users easier. The ```.whl``` format is the default preference for ```pip``` when you run a ```pip install``` command.
+
+OpenVino wheels for various platforms are available through [PyPI](https://pypi.org/project/openvino/#files) and allow you to install OpenVINO™ in either your Python distribution or a dedicated virtual environment.
+
+OpenVino builds two different wheels with separate ```setup.py``` files for [```openvino```](../wheel/setup.py) and [```openvino-dev```](../../../../tools/openvino_dev/setup.py). To build the wheels while building the project from source, your ```cmake``` command must include ```DENABLE_PYTHON=ON``` and ```DENABLE_WHEEL=ON```. Once built, the wheels should be available under ```openvino_install_dir/tools```
```
Comment (ilya-lavrenov) [inline src/bindings/python/docs/build.md:11]:
```suggestion
OpenVino builds two different wheels with separate ```setup.py``` files for [```openvino```](../wheel/setup.py) and [```openvino-dev```](../../../../tools/openvino_dev/setup.py). To build the wheels while building the project from source, your ```cmake``` command must include ```-DENABLE_PYTHON=ON``` and ```-DENABLE_WHEEL=ON```. Once built, the wheels should be available under ```openvino_install_dir/tools``` 
```

**Review by kblaszczak-intel — APPROVED** [review]

**Comment by kblaszczak-intel** [comment]
Since all comments have been resolved, can we merge this one?

**Comment by kblaszczak-intel** [comment]
@jagadeeshmadinni Is this ready for publishing? Can we try to merge this one?

**Comment by jagadeeshmadinni** [comment]
> @jagadeeshmadinni Is this ready for publishing? Can we try to merge this one?

Yes, please.

**Pull request merged**